datahub-web-react/src/app/ingest/source/builder/RecipeForm/unity_catalog.tsx

```diff
@@ -0,0 +1,153 @@
+            Enabling this feature may increase the duration of ingestion.
+        </div>
+    ),
+    type: FieldType.BOOLEAN,
+    fieldPath: 'source.config.include_column_lineage',
+    rules: null,
+};
+
+const metastoreIdAllowFieldPath = 'source.config.metastore_id_pattern.allow';
+export const UNITY_METASTORE_ID_ALLOW: RecipeField = {
+    name: 'metastore_id_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip:
+        'Only include specific Metastores by providing the id of a Metastore, or a Regular Expression (REGEX) to include specific Metastores. If not provided, all Metastores will be included.',
+    placeholder: '11111-2222-33333-44-555555',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: metastoreIdAllowFieldPath,
+    rules: null,
+    section: 'Metastores',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, metastoreIdAllowFieldPath),
+};
+
+const metastoreIdDenyFieldPath = 'source.config.metastore_id_pattern.deny';
+export const UNITY_METASTORE_ID_DENY: RecipeField = {
+    name: 'metastore_id_pattern.deny',
+    label: 'Deny Patterns',
+    tooltip:
+        'Exclude specific Metastores by providing the id of a Metastores, or a Regular Expression (REGEX). If not provided, all Metastores will be included. Deny patterns always take precedence over Allow patterns.',
+    placeholder: '11111-2222-33333-44-555555',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: metastoreIdDenyFieldPath,
+    rules: null,
+    section: 'Metastores',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, metastoreIdDenyFieldPath),
+};
+
+const catalogAllowFieldPath = 'source.config.catalog_pattern.allow';
+export const UNITY_CATALOG_ALLOW: RecipeField = {
+    name: 'catalog_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip:
+        'Only include specific Catalogs by providing the name of a Catalog, or a Regular Expression (REGEX) to include specific Catalogs. If not provided, all Catalogs will be included.',
+    placeholder: 'my_catalog',
+    type: FieldType.LIST,
+    buttonLabel: 'Add pattern',
+    fieldPath: catalogAllowFieldPath,
+    rules: null,
+    section: 'Catalogs',
+    setValueOnRecipeOverride: (recipe: any, values: string[]) =>
+        setListValuesOnRecipe(recipe, values, catalogAllowFieldPath),
+};
+
+const catalogDenyFieldPath = 'source.config.catalog_pattern.deny';
+export const UNITY_CATALOG_DENY: RecipeField = {
+    name: 'catalog_pattern.allow',
+    label: 'Allow Patterns',
+    tooltip:
+        'Exclude specific Catalogs by providing the name of a Catalog, or a Regular Expression (REGEX) to exclude specific Catalogs. If not provided, all Catalogs will be included. Deny patterns always take precedence over allow patterns.',
```

Comment:
Can we capitalize Allow here for the sake of consistency? Same elsewhere in the file

Reply:
Yes!